packages/shared/src/hooks/useFeedSettings.ts

```diff
@@ -42,10 +42,13 @@ export default function useFeedSettings(): FeedSettingsReturnType {
       return;
     }
 
-    requestIdleCallback(() => {
-      queryClient.invalidateQueries(generateQueryKey('popular', user));
-      queryClient.invalidateQueries(generateQueryKey('recent', user));
-    });
+    requestIdleCallback(
+      () => {
+        queryClient.invalidateQueries(generateQueryKey('popular', user));
+        queryClient.invalidateQueries(generateQueryKey('recent', user));
+      },
+      { timeout: 100 },
```

Comment:
This option actually doesn't do what we want.
It limits this call to be within a 100ms window.
What we want is to call is this function after 100ms. So we need to change it to `setTimeou` instead of idle callback